oca_monetize/static/src/js/oca_monetize.js

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2024 Hunki Enterprises BV
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0) */
+
+odoo.define("oca_monetize", function (require) {
+    "use strict";
+
+    var core = require("web.core");
+    var rpc = require("web.rpc");
+
+    core.bus.on("web_client_ready", null, function () {
```

Comment:
I think a warning appearing for all users, perpetually, is perhaps too much nag for people to live with on a daily basis. CEO's will complain that their users get confused by it, it takes space on the screen, etc.

I think the message should be:

- Directed at the higher end users, so maybe the ones who have Administrator rights only - these should normally be the ones deciding what to pay for
- Temporary - eg like Wikipedia has these temporary funding runs, in order not to affect usability so much
- Friendly - we're asking / trying to convince, not requiring

What I thought really sympathetic and what made me pay is how the creator of KeePass for Android pops up a message after each Oktoberfest has passed, asking to buy him a beer at the Octoberfest if you like the app. If you don't like the app enough, by all means don't do it.

This serves to not annoy people, eg you can easily click the message away, but even a yearly reminder serves already the purpose of making people aware that there is a team of volunteers behind this that created a specific value for you, and explains you how you can contribute. That's already a win compared to the current situation of most people being completely unaware.